generatorv2/README.md

```diff
@@ -0,0 +1,134 @@
+# Generator
+
+Last Updated: 2024/11/9
+
+The generator is a powerful tool that can be used to generate tests for exercises based on the canonical data.
+The generator is written in Ruby and is located in the `bin` directory.
+
+## How to use the generator
+
+### Things to do before running the generator
+
+Run `bundle install` to install the required libraries.
+Before running the generator you have to make sure a couple of files are in place.
+
+1. `tests.toml` file
+
+It is located under the `.meta` folder for each exercise.
+The toml file is used to configure which exercises are generated and which are not.
+Since the generator grabs all the data from the canonical data, so does this enable new tests that won't automatically be merged in.
+Instead so does new tests have to be added to the toml file before they show up in the test file.
+
+If there is a test that isn't needed or something that doesn't fit Ruby you can remove it from the toml file.
```

Comment:
```suggestion
If there is a test that isn't needed or something that doesn't fit Ruby you can remove it from the configuration file.
```

In case we move from TOML to another type of configuration file, this reference will not need to change in the documentation.